frontend/src/modules/organization/components/organization-identities.vue

```diff
@@ -67,7 +67,7 @@ const platforms = computed(() => [...new Set(props.organization.identities.map((
 
 const getHandlesByPlatform = (platform) => props.organization.identities
   .filter((i) => i.platform === platform)
-  .map((i) => (i.url ? i.url.split('/').at(-1) : i.name));
+  .map((i) => getPlatformDetails(i.platform).organization.handle(i));
```

Comment:
We should protect this for custom platforms. I think for custom platforms `getPlatformDetails(i.platform)` this will return undefined and it will break and won't get the organization handle.

Should it be 
```
getPlatformDetails(i.platform)?.organization.handle(i)) || i.name
```